static/sass/styles.scss

```diff
@@ -346,3 +346,8 @@ h2 {
   opacity: 1 !important;
   transition: opacity 0.5s ease-in;
 }
+
+// Util helper to remove borders
```

Comment:
It might be nice if we set up a standard practice across the projects for how we deal with overrides or utils, like creating a separate file or something.